### PR TITLE
feat: no pendo agent fixups

### PR DESCRIPTION
**Pull request by 0xdef1cafe**
## Description

bypasses fixing up the agent, just load from src - we're happy to trust pendo

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

n/a

## Risk

if pendo is compromised and delivers a malicious payload native wallet users may lose funds.

## Testing

* pendo guides should work
* to be tested by @DiggyDiggy2 @reallybeard in `release` environment - `develop` has special env vars set to enable "unsafe designer mode", and `release` mirrors production. we want guides to work in production.

### Engineering

* see if you can find more code to delete

### Operations

* no testing required by ops

## Screenshots (if applicable)

no visual changes

### CODE REVIEW

File: src/plugins/pendo/agent/arm.ts [hunk 1]
```diff
@@ -61,43 +59,31 @@ export function armPendoAgent(
...
-    ).text()
+    const originalAgent = await fetch(agentUrl, {
+      credentials: 'omit',
+    })
+
+    const agentSrc = await originalAgent.text()
+
     const agentIntegrity = (await ssri.fromData(agentSrc, { algorithms: ['sha256'] })).toString()
     moduleLogger.trace({ fn: 'armPendoAgent', agentIntegrity }, 'parsing agent')
-    const parsedAgent = await parseAgent(agentSrc, pendoConfig, fixupTables)
```
Comment (0xdef1cafe) [inline src/plugins/pendo/agent/arm.ts:71]: the magic - don't molest the agent, just stick it in the page

**Review by 0xApotheosis — APPROVED** [review]
Looks sane to me - couldn't find any obvious things we could additionally remove.

Will leave product to confirm functionality still as expected.

**Review by gomesalexandre — APPROVED** [review]
Conceptual stamp - happy with this one if product is happy

**Comment by mrnerdhair** [comment]
For reference, this wasn't done to avoid trusting Pendo's code specifically -- it's about trusting anyone who has access to the Pendo designer (including, for example, someone that hacks a product person's account -- last I checked Pendo doesn't do 2FA!) with the ability to inject code into pages on prod. Code that's deployed maliciously that way would bypass all the usual engineering safeguards, could be targeted to never appear on machines heuristically identified as potential developers' machines, and it would be very difficult to track down and identify such an attacker post-mortem.

It's also about data leakage; without explicit filtering, Pendo will be happy to collect wallet addresses/balances/etc, and nobody's really going to notice unless they're explicitly looking through the collected events for them.

I advise against accepting these risks.

**Pull request merged**